variables.tf

```diff
@@ -812,6 +817,12 @@ variable "rds_subnet_tag_filter" {
   default     = {}
 }
 
+variable "rds_subnet_cidr_block_filter" {
```

Comment:
Please sepcify the type of variable explicitely

Reply:
Done with [6232c30](https://github.com/Flaconi/terraform-aws-microservice/pull/62/commits/6232c305a6045017894e7f7d13a9f0bd647577d7).

Reply:
> Please sepcify the type of variable explicitely

Something that for us (DevOps team members) is seen as common sense, for others might not be entirely obvious. I'll open a PR here: https://github.com/Flaconi/devops-docs/blob/master/doc/conventions/style-guide-terraform.md